tests/operations.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitAndStartInstance,
+  visitAndStopInstance,
+} from "./helpers/instances";
+import { assertTextVisible } from "./helpers/permissions";
+import {
+  checkNotificationExists,
+  dismissNotification,
+} from "./helpers/notification";
+import { validateOperation } from "./helpers/operations";
+
+let instance = randomInstanceName();
```

Comment:
```suggestion
const instance = randomInstanceName();
```
The linter complained about this, it can be a const and it can be within the test, no need to have it on the root level.